### ISSUE TITLE
No infotext for entities in F5 mode

### ISSUE THREAD

**Issue by Fixer-007** [minetest/minetest#5042]
There is no infotext for entities in F5 mode, example:

![b0ildql](https://user-images.githubusercontent.com/3686677/47945735-eda57000-defb-11e8-9666-7de336179ade.png)

I see debug infotext of dropped stuff but without its name:name and description name.

I think it was intended to show that info in debug mode when this https://github.com/minetest/minetest/pull/4134 was discussed but it seems it was left out.

My suggestion is to print name:name and description name above that GenericCAO hp=1 infotext.

@WSDGuy2014 @paramat and other interested are welcome

**Comment by DS-Minetest** [comment]
It would be nice to have the name of the pointed entity where the pointed node names are normally.

**Comment by Fixer-007** [comment]
Here is how it looks now:
![default](https://cloud.githubusercontent.com/assets/16494741/23959916/299c868a-09af-11e7-967b-905c7e551b48.png)

It seems trivial to fix, can anyone look into this? Thanks

**Comment by paramat** [comment]
Can't you define entity infotext in entity definition?
https://github.com/minetest/minetest/blob/79e393cf4b5027eb9071df23f4834aac3f3a689f/doc/lua_api.txt#L5545

**Comment by SmallJoker** [comment]
> My suggestion is to print name:name and description name above that GenericCAO hp=1 infotext.

This would only work for players where the name is sent - for LuaEntities it's just an empty string:
https://github.com/minetest/minetest/blob/79e393cf4b5027eb9071df23f4834aac3f3a689f/src/content_sao.cpp#L529
Thing is, even the server does only barely know about what the entity actually is. They're all kept indexed by integers, and callbacks are executed using a (more or less) private table from builtin, which then keeps the entity definition in a (as already expected) integer-indexed array to directly look them up and call from C++.
So this PR not only involves some changes lines in the debug display, but also changes in how entities are registered.

TL;DR: Server does not send enough info to implement this yet.

**Comment by paramat** [comment]
Any core dev support for working towards this?
